spec/views/plantings/edit.html.haml_spec.rb

```diff
@@ -40,7 +40,6 @@
 
     it 'includes helpful links for crops and gardens' do
```

Comment:
You're probably right about both things, but because view tests are deprecated I generally only change them just enough to prevent failing.